README.md

```diff
@@ -90,9 +90,9 @@ def mystery5(s)
 end
 ```
 
-- What is mystery5("hi")?
-- What is mystery5("")?
-- What is mystery5("Hi, there!")?
+- What is mystery5("hi")? **
+- What is mystery5("")? ""
+- What is mystery5("Hi, there!")? **********
```

Comment:
You should always put quotations around results that are strings. 
"**"
""
"**********"